plugins/MCWhitelist/__init__.py

```diff
@@ -0,0 +1,192 @@
+import discord
+from discord.ext import commands
+
+from Plugin import AutomataPlugin
+from Globals import (
+    mongo_client,
+    PRIMARY_GUILD,
+    VERIFIED_ROLE,
+    EXECUTIVE_ROLE,
+)
+from .mojang_api import MojangAPI
+from .whitelist_http_api import WhitelistHttpApi
+
+
+def is_executive():
```

Comment:
Elsewhere we use a permission check rather than a custom predicate. You can see this in use for the identity check function. I'd suggest either adopting that approach, or putting this predicate in a shared file and converting that to use that as well